temba/flows/models.py

```diff
@@ -1017,35 +1014,6 @@ def get_versions_after(cls, version_number):
     def as_select2(self):
         return dict(id=self.uuid, text=self.name)
 
-    def release(self):
```

Comment:
Don't we need any more of this method?

Reply:
We don't need, because this method is overwritten by 
https://github.com/greatnonprofits-nfp/rapidpro/blob/32b86a5ff78128310500512a51374c687d2ee4a8/temba/flows/models.py#L2133-L2164